bonding.py

```diff
@@ -28,7 +28,7 @@
 from optparse import OptionParser, OptionGroup
 from distutils.version import LooseVersion
 
-__version__ = '1.0.0'
+__version__ = '1.1.0'
```

Comment:
I thought there was a way to cherry pick in github so I did a separate commit for the version bump so you could skip it... but the Googles they do nothing.